logic.py

```diff
@@ -21,14 +23,32 @@ def tuple2list(ExTuple):
     return ExList 
 
 def SQLQuery(statment):
-    cursor.execute(statment)
-    conn.commit()
-    data = logic.tuple2list(cursor.fetchall()) 
-    return data
+    try:
+        cursor.execute(statment)
+        conn.commit()
+        data = logic.tuple2list(cursor.fetchall())
+        status = "OKAY"
+        purpose = "PRODUCTION"
+        SQL_log(statment,status,purpose)
+        return data
+    except:
+        status = "ERROR"
```

Comment:
Is there a way that we can define what type of error is it?

Reply:
I have done some research and i am unable to figure it out. If we see this is a much needed feature, we can return to it in later sprints

Reply:
Alright, if this isn't a much-needed feature then this should work. However, we can discuss this in the later sprint.